package.json

```diff
@@ -53,5 +53,8 @@
     "*.{json,md}": [
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "rollup-plugin-esbuild": "2.0.0"
```

Comment:
this should be on cli though, right?

Reply:
Yep

Reply:
Yep, was pretty sure I ran it in the cli package, gonna move it there